### PR TITLE
Update docstring and add pytest 

### LINKED ISSUE #85: Test fileHandler

### PR DESCRIPTION
**Pull request by awirb**
* update docstrings 
* add pytest for fileHandlerUtils fix #85

### CODE REVIEW

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 """
-    This is a python wrapper to execute a compiled com1Exe file
-
-    This file is part of Avaframe.
+    This is a python wrapper to execute the compiled com1Exe file and set desired simulation options
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:2]:
```suggestion
    Python wrapper to execute the compiled com1Exe file and set desired simulation options
```

File: avaframe/in3Utils/initialiseDirs.py [hunk 2]
```diff
@@ -21,7 +21,22 @@
 
 
 def initialiseRunDirs(avaDir, modName):
-    """ Initialise Simulation Run with input data """
+    """ Initialise Simulation Run with input data
```
Comment (fso42) [inline avaframe/in3Utils/initialiseDirs.py:24]:
```suggestion
    """ Initialise simulation run with input data
```

**Review by fso42 — APPROVED** [review]
just spelling issues

File: avaframe/com1DFA/com1DFA.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:2]: Done

File: avaframe/in3Utils/initialiseDirs.py (hunk 2 above)
Comment (awirb) [reply avaframe/in3Utils/initialiseDirs.py:24]: Done

**Pull request merged**